### ISSUE TITLE
CFL and BKG detector group trigger corrections index off by one

### ISSUE THREAD

**Issue by samaloney** [i4Ds/STIX-GSW#80]
**Short Description**
During a discussion with @paolomassa we found there was an off by one error in the indices used to correct the triggers CFL and BKG detector group was off by one. Will probably be addressed as part of #49 but just writing down here incase it's forgotten.

**Details**
Current
```
;triggers
t2det=stx_ltpair_assignment(dindgen(32)+1,/adgroup_idx)

bkg_ratio = 1.1
cfl_ratio = 1.33
triggers32=data.triggers(t2det-1)
;CFL and det8
triggers32(8,*)=data.triggers(t2det(8))/(1.+cfl_ratio)*cfl_ratio
triggers32(7,*)=data.triggers(t2det(7))/(1.+cfl_ratio)
;BKG and det 16
triggers32(9,*)=data.triggers(t2det(9))/(1.+bkg_ratio)*bkg_ratio
triggers32(15,*)=data.triggers(t2det(15))/(1.+bkg_ratio)
```

Should be
```
;triggers
t2det=stx_ltpair_assignment(dindgen(32)+1,/adgroup_idx)

bkg_ratio = 1.1
cfl_ratio = 1.33
triggers32=data.triggers(t2det-1)
;CFL and det8
triggers32(8,*)=data.triggers(t2det(8)-1)/(1.+cfl_ratio)*cfl_ratio
triggers32(7,*)=data.triggers(t2det(7)-1)/(1.+cfl_ratio)
;BKG and det 16
triggers32(9,*)=data.triggers(t2det(9)-1)/(1.+bkg_ratio)*bkg_ratio
triggers32(15,*)=data.triggers(t2det(15)-1)/(1.+bkg_ratio)
```
or even

```
triggers32(8,*)/=(1.+cfl_ratio)*cfl_ratio
triggers32(7,*)/=(1.+cfl_ratio)
;BKG and det 16
triggers32(9,*)/=(1.+bkg_ratio)*bkg_ratio
triggers32(15,*)/=/(1.+bkg_ratio)
```
**Environment**
- IDL Version:
- STIX GSW Version: master
- OS:

**Comment by paolomassa** [comment]
Yes, I corrected it and my pull request has already been merged

**Comment by grazwegian** [comment]
Closed by #89
